src/views/Workflow.vue

```diff
@@ -175,9 +201,57 @@ export default {
             }
           })
       }
+      if (this.deltaSubscriptions.length === 0) {
+        const vm = this
+        this.$workflowService
+          .startDeltasSubscription(WORKFLOW_TREE_DELTAS_SUBSCRIPTION, this.variables, {
+            next: function next (response) {
+              applyDeltas(response.data.deltas, vm.tree)
+              vm.isLoading = false
+            },
+            error: function error (err) {
+              vm.setAlert(new Alert(err.message, null, 'error'))
+              vm.isLoading = false
+            }
+          })
+      }
       this.deltaSubscriptions.push(id)
       return id
     },
+    subscribeTableDeltas () {
+      const id = new Date().getTime()
+      // start deltas subscription if not running
+      if (this.deltaTableSubscriptions.length === 0) {
+        const vm = this
+        this.$workflowService
+          .startDeltasSubscription(WORKFLOW_TABLE_DELTAS_SUBSCRIPTION, this.variables, {
+            next: function next (response) {
+              applyTableDeltas(response.data.deltas, vm.table)
+              vm.isLoading = false
+            },
+            error: function error (err) {
+              vm.setAlert(new Alert(err.message, null, 'error'))
+              vm.isLoading = false
+            }
+          })
+      }
+      if (this.deltaTableSubscriptions.length === 0) {
+        const vm = this
+        this.$workflowService
+          .startDeltasSubscription(WORKFLOW_TABLE_DELTAS_SUBSCRIPTION, this.variables, {
+            next: function next (response) {
+              applyTableDeltas(response.data.deltas, vm.table)
+              vm.isLoading = false
+            },
+            error: function error (err) {
+              vm.setAlert(new Alert(err.message, null, 'error'))
+              vm.isLoading = false
+            }
+          })
+      }
```

Comment:
The code above also has a bug. The `if` blocks are identical, probably happened due to copy-and-paste, or during git/rebase/merge/etc

Reply:
Thank you - duplicate `if` code block removed.